examples/chemistry/brusselator.jl

```diff
@@ -91,31 +73,23 @@ V = map(sd[:point]) do (x,_)
   27 * (x *(1-x))^(3/2)
 end
 
-# TODO: Try making this sparse.
 F₁ = map(sd[:point]) do (x,y)
  (x-0.3)^2 + (y-0.6)^2 ≤ (0.1)^2 ? 5.0 : 0.0
 end
-mesh(s, color=F₁, colormap=:jet)
 
-# TODO: Try making this sparse.
 F₂ = zeros(nv(sd))
 
-One = ones(nv(sd))
-
 constants_and_parameters = (
-  fourfour = 4.4,
-  threefour = 3.4,
   α = 0.001,
   F = t -> t ≥ 1.1 ? F₂ : F₁)
 
 # Generate the simulation.
-gensim(expand_operators(Brusselator))
-sim = eval(gensim(expand_operators(Brusselator)))
-fₘ = sim(sd, generate)
```

Comment:
I think this defaults to `GeometricHodge` so since we use `Circumcenter` subdivision we should set this to be `DiagonalHodge.